aten/src/ATen/native/cuda/BatchLinearAlgebra.cu

```diff
@@ -32,7 +32,7 @@ void magmaSolveBatched(
     magma_int_t* dinfo_array, magma_int_t batch_count, const MAGMAQueue& magma_queue);
 
 template<class scalar_t>
-void magmaLu(
+void cu_xgetrf(
```

Comment:
I would prefer if you retained the naming scheme: for example - `cuSolverLu` for consistency.

Reply:
Will fix